ipaclient/install/ipa_epn.py

```diff
@@ -238,12 +238,35 @@ def add_options(cls, parser):
 
     def validate_options(self):
         super(EPN, self).validate_options(needs_root=True)
-        if self.options.to_nbdays:
+        if self.options.to_nbdays is not None:
+            try:
+                int(self.options.to_nbdays)
+                assert float(self.options.to_nbdays) == \
+                    int(self.options.to_nbdays)
```

Comment:
You don't want to use assert here, instead if x != y: raise exception.

What is it you're trying to verify? That a fraction isn't used, like 3.3?

Reply:
Yes, it was a quick hack to check for decimal input.

Reply:
Good catch, @flo-renaud thanks!